lib/handlers/get.js

```diff
@@ -76,7 +77,15 @@ function handler (req, res, next) {
     // Till here it must exist
     if (!includeBody) {
       debug('HEAD only')
-      res.sendStatus(200)
+      const rootUrl = utils.getBaseUri(req)
+      const mapper = new LegacyResourceMapper({
```

Comment:
Just wondering if you can initialize this earlier and reuse; fine otherwise.

Reply:
I could probably do that, this is more or less a hack to get it handled properly